db/migrate/20240627161450_add_all_credential_emails_to_o_auth_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllCredentialEmailsToOAuthSessions < ActiveRecord::Migration[7.1]
+  def change
+    add_column :oauth_sessions, :all_credential_emails, :string, array: true, default: []
```

Comment:
Could you remove this migration? I think you broke this out into a separate PR

Reply:
Yep, I'll remove it once people are done testing